style-editor.css

```diff
@@ -1,17 +1,29 @@
-
 /*--------------------------------------------------------------
 # Editor Styles 
 #
 # This is the place to enter any custom overrides or structural 
 # styles for the block editor.
 --------------------------------------------------------------*/
 
+/*
+This multiplier is to account for the changes in width between the editor and the front end.
```

Comment:
Maybe it's just me, but scaling the type sizes between the editor and front-end seems counterintuitive. As a content creator, I would expect that the size I'm being shown in the editor is what appears on my site, especially if you're giving me a way to set the type size explicitly via the block settings. But I may be biased here because I spend all day trying to make themes where the editor experience mirrors the front end 😅.

Also, the `has-x-font-size` rules aren't respected when you use the change block's font size via "Text Settings" due to the styles being inlined:

<img width="1792" alt="Screen Shot 2020-05-12 at 10 09 08 PM" src="https://user-images.githubusercontent.com/5375500/81763480-58464d00-949d-11ea-8763-9ac4d72de5ff.png">

Reply:
I disabled custom header sizes in b2323aa4caf05d2d5692838b383d16143a4698c4, so that should take care of inlining.

Regarding mirroring the experience, I'd love to have things be identical, but I've seen a lot of issues with Gutenberg when the code sets the editor's width to 700px. Is that something you know how to work around?

Reply:
So I re-read the Gutenberg theming guide, and noticed there's https://developer.wordpress.org/block-editor/developers/themes/theme-support/#changing-the-width-of-the-editor. I implemented that in 3349d3, so we should be good here!